typescript/infra/config/environments/mainnet3/warp/configGetters/formConfigGetters.ts

```diff
@@ -0,0 +1,65 @@
+import { ethers } from 'ethers';
+
+import { ChainMap, HypTokenRouterConfig, TokenType } from '@hyperlane-xyz/sdk';
+import { Address } from '@hyperlane-xyz/utils';
+
+import {
+  RouterConfigWithoutOwner,
+  tokens,
+} from '../../../../../src/config/warp.js';
+
+const safeOwners: ChainMap<Address> = {
+  ethereum: '0xec5ad23e29203301B2C1a765718Cc1de7A8d3FbF',
+  form: '0x41B624412B529409A437f08Ef80bCabE81053650',
+};
+
+export const getEthereumFormUSDCWarpConfig = async (
```

Comment:
would probably prefer us to split out these into their own files to stick with the pattern. We can ofc import the owners from one of them into the other